debian/marquee/nginx/sites/html.spec.whatwg.org.conf

```diff
@@ -26,6 +26,9 @@ server {
     location = /multipage/web-sockets.html {
         return 301 https://websockets.spec.whatwg.org/;
     }
+    location = /multipage/websockets {
```

Comment:
Running the tests after deploy reveals that this redirects https://html.spec.whatwg.org/multipage/websockets but not https://html.spec.whatwg.org/multipage/websockets/.

Reply:
I fixed this in https://github.com/whatwg/misc-server/commit/a9ab4ed7d9de6a02c17cade32df0ae1dc632673a and tests are now passing.